tests/test_other.py

```diff
@@ -225,7 +225,7 @@ def test_emcc_2(self):
           print('(no output)')
           continue
         syms = Building.llvm_nm(target)
-        assert len(syms.defs) == 1 and 'main' in syms.defs, 'Failed to generate valid bitcode'
+        assert len(syms.defs) == 1 and ('main' in syms.defs or '__original_main' in syms.defs), 'Failed to generate valid bitcode'
```

Comment:
I'm concerned this was needed since `main` should always exist, even when `__original_main` exists too.

Reply:
Thanks, that is indeed a bug, fixing.